### PR TITLE
[legacy audit] add response audit for hijack

### LINKED ISSUE #47097: [legacy audit] "kubectl exec" doesn't have response line in audit
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG

**What happened**:
When run "kubectl exec", there is only on request line in legacy audit.

**What you expected to happen**:
Just like other request, there should be one request line and one response line.

**How to reproduce it** (as minimally and precisely as possible):
run "kubectl exec xxx" and refer to legacy audit.

### PR DESCRIPTION
**Pull request by CaoShuFeng**
Fixes #47097
**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

File: staging/src/k8s.io/apiserver/pkg/endpoints/filters/legacy_audit.go [hunk 1]
```diff
@@ -43,12 +43,15 @@ type legacyAuditResponseWriter struct {
 	id  string
 }
 
-func (a *legacyAuditResponseWriter) WriteHeader(code int) {
+func (a *legacyAuditResponseWriter) formatResponse(code int) {
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/pkg/endpoints/filters/legacy_audit.go]: This is actually `printResponse`?

**Comment by deads2k** [comment]
@soltysh @sttts any reason I shouldn't merge this after the function name is fixed?

**Comment by sttts** [comment]
lgtm

**Comment by deads2k** [comment]
@CaoShuFeng  please open an issue that I can tag to merge into 1.7.

**Comment by CaoShuFeng** [comment]
@deads2k Thanks for your review.
OK, I will do it tomorrow(about 12 hours later).

File: staging/src/k8s.io/apiserver/pkg/endpoints/filters/legacy_audit.go (hunk 1 above)
Comment (CaoShuFeng) [inline staging/src/k8s.io/apiserver/pkg/endpoints/filters/legacy_audit.go]: Done.

**Comment by CaoShuFeng** [comment]
@deads2k the issue: #47097

**Pull request merged**